modules/tinymce/src/themes/silver/test/ts/browser/editor/ShowHideTest.ts

```diff
@@ -35,10 +37,15 @@ describe('browser.tinymce.themes.silver.editor.ShowHideTest', () => {
     ]
   });
 
+  const pFocusEditor = async (editor: Editor) => {
```

Comment:
This name is very misleading, as it's doing much more than just focusing the editor. Maybe `pActivateEditor`, `pShowToolbar` or something like that?